src/main/java/frc/robot/subsystems/snapback/shooter/ShooterIO.java

```diff
@@ -17,12 +17,16 @@ public static class ShooterIOInputs {
     public double leftAppliedVolts;
     public double leftCurrentAmps;
     public double leftTemperatureCelsius;
+    public double leftVelocitySetpointRadiansPerSecond;
+    public double leftVelocityErrorRadiansPerSecond;
 
     public Rotation2d rightPosition;
     public double rightVelocityRadiansPerSecond;
     public double rightAppliedVolts;
     public double rightCurrentAmps;
     public double rightTemperatureCelsius;
+    public double rightVelocitySetpointRadiansPerSecond;
```

Comment:
We need an input for the goal velocity:

```suggestion
    public double rightVelocityGoalRadiansPerSecond = 0.0;
```